portal/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -11,3 +11,5 @@ final case class Project(
 )
 
 final case class ProjectId(value: String) extends MappedTo[String]
+
+final case class AddProjectRequest(ownerId: UserId, name: String, repository: String)
```

Comment:
case class names should not be verbs, ProjectAdditionRequest (or some other noun) is much more suitable

Reply:
renamed